### PR TITLE
Project for Code review

### PR DESCRIPTION
**Pull request by sja-thedude**
-Html File has Been Added
-CSS File has Been Added
-Linters File has Been Added
-Readme File has Been Added

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@media (max-width: 460px){
+  h1 {
+    font-size: 3.5rem;
+    line-height: 1.2;
+  }
+  #main {
```
Comment (sja-thedude) [inline css/style.css]: Please check and fix the linter errors in the code.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  height: auto;
+  margin: 0 auto;
+}
+
+#img-div {
+  margin: 0 auto;
+  width: 90%;
+}
+
+img {
```
Comment (sja-thedude) [inline css/style.css]: Please check and fix the linter error in the code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding
+          pest-resistent plants.
+        </li>
+      </ul>
+    </section>
+    <div class="footer">
+      <h2>
+        If you have time, you should read more about this incredible human being on his <a id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>.
+      </h2>
```
Comment (sja-thedude) [inline index.html]: Please fix the linter error.
Link 'rel' attribute should include 'noopener' or 'noreferrer'.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <div id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="">
+      <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+        increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </div>
```
Comment (sja-thedude) [inline index.html]: Please fix the linter error. Use semantic tags. Element “figcaption” is not allowed as a child of element “div” in this context.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
```
Comment (sja-thedude) [inline index.html:5]: Kindly add the 'viewport' meta element which was not specified. Fix Linter error.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+          <span class="b">1937</span> - Finishes university and takes a job in the US Forestry Service
+        </li>
+        <li class="tribute-li">
+          <span class="b">1938</span> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts.
+          Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding
+          pest-resistent plants.
+        </li>
+      </ul>
+    </section>
+    <div class="footer">
```
Comment (sja-thedude) [inline index.html:48]: It's better to use the semantic tag instead of div.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (sja-thedude) [inline css/style.css:47]: Kindly use a professional way to write your commits.

**Review by sja-thedude — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re-review when you're done.

Thanks 😀

**Review by afanimartin — APPROVED** [review]
Hi @sja-thedude, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**